web-ui/src/main/resources/catalog/components/search/map/MapFieldDirective.js

```diff
@@ -134,20 +134,253 @@
               var updateField = function(geom) {
                 feature.setGeometry(geom);
 
-                // Write the extent as 4326 WKT polygon
-                var lonlatFeat, writer, wkt;
+                // Get projection code of current map view
+                var proj = scope.map.getView().getProjection().getCode();
+
+                /**
+                 * Transforms the given longitude-latitude pair to XY coordinates in the current projection.
+                 * @param {number} lon 
+                 * @param {number} lat 
+                 */
+                var getXY = function(lon, lat) {
+                  return (new ol.geom.Point([lon, lat])).transform('EPSG:4326', proj).getFirstCoordinate();
+                }
+
+                // Define polar coordinates in current projection
+                var southPole = getXY(0, -90);
```

Comment:
this can be written as `ol.proj.fromLonLat([0, -90], proj)`

Reply:
Ah, was looking for that, thanks. Will change.